pkg/cluster/controllers/database/readiness.go

```diff
@@ -49,11 +51,25 @@ func (p *PostgreSQLController) Readiness(ctx context.Context) (*lcm.CRStatus, er
 
 	name := p.HarborCluster.Name
 
-	conn, _, err = p.GetInClusterDatabaseInfo(ctx)
+	conn, err = p.GetInClusterDatabaseInfo()
```

Comment:
Why do we still need to get conn as we don't check health with that client anymore?